here/traffic/sql/create_here_valid_range.sql

```diff
@@ -0,0 +1,23 @@
+-- View that documents which street versions correspond to 
+-- which date range in here.ta table 
+
+CREATE TABLE here.street_valid_range AS 
+
+WITH street_v(street_version, valid_range) AS (
+	VALUES 
+	('18_3', daterange('2014-01-01', '2017-01-01')),
+	('19_4_tc', daterange('2017-01-01', '2017-09-01')),
+	('21_1', daterange('2017-09-01', '2022-08-15')),
+	('22_2', daterange('2022-08-15', 'infinity'))
```

Comment:
Better to keep the upper range on this one undefined. https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-INFINITE

Should be the same effect, but clearly indicates that this is a (currently) unknown value.